### ISSUE TITLE
Evaluate & specify TurtleBot - FMS interface 

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#18]
## User story
1. As a SD
2. I want to brainstorm on the interface between TurtleBot and fleet management software
3. So that I have a better idea on how these components communicate.
[VDA5050-V2_0_0-DE.pdf]
(https://github.com/amosproj/amos2022ss03-turtlebot-fleet-management/files/8667729/VDA5050-V2_0_0-DE.pdf)

## Acceptance criteria
* Good understanding of the interface between specified components. 
* Documenting a standard on how the components communicate, aka create API docs.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Sebastian2023** [comment]
The TurtleBot side of the interface also looks good. There are already ready packages for the MQTT bridge and the VDA5050 messages.

https://github.com/ipa320/vda5050_msgs
